components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -116,11 +117,17 @@ IJsonSerializer jsonSerializer
 
         /// <summary>
         /// Receives the heartbeat from the LegacyJobOrchestrator, JobProcess; used to control lifetime of job (cancel, restart, keep).
+        /// Used also to receive the diagnostic info
         /// </summary>
         /// <param name="heartbeat"></param>
+        /// <param name="info"></param>
         /// <param name="ct"></param>
         /// <returns></returns>
-        public async Task<HeartbeatResultModel> SendHeartbeatAsync(HeartbeatModel heartbeat, CancellationToken ct = default) {
+        public async Task<HeartbeatResultModel> SendHeartbeatAsync(
+            HeartbeatModel heartbeat, 
+            JobDiagnosticInfoModel info, 
```

Comment:
Can we please call it `diagInfo` or something like that so that the name is more self-explanatory ?

Reply:
renamed